src/components/Chips/Chips.tsx

```diff
@@ -129,86 +129,96 @@ const Chips: VibeComponent<ChipsProps, HTMLElement> & {
       [onClick]
     );
 
+    const overrideClassName = cx(styles.chips, "chips--wrapper", className, {
+      [styles.disabled]: disabled,
+      [styles.withClose]: hasCloseButton,
+      [styles.noAnimation]: noAnimation,
+      [styles.withUserSelect]: allowTextSelection
+    });
+
+    const clickableProps = useClickableProps(
+      {
+        onClick: onClickCallback,
+        onMouseDown,
+        className: overrideClassName,
+        enableTextSelection: allowTextSelection,
+        disabled,
+        id,
+        dataTestId: overrideDataTestId,
+        ariaLabel: ariaLabel || label,
+        ariaHidden: false,
+        ariaHasPopup: false,
+        ariaExpanded: false,
+        style: backgroundColorStyle
```

Comment:
not need to be part of hook because hook is logical only and indifferent to style settings

Reply:
removed from the hook